interpreter.py

```diff
@@ -1,46 +1,68 @@
 from __future__ import with_statement
-
-import sys
-
 from rpython.rlib import jit
 from llvm_wrapper import *
+from type_wrapper import *
```

Comment:
please don't use ``import *``, it makes reading the code much harder ("where does this class come from?")